File: AndroidStudio_folder/ch7Quiz/app/src/main/java/com/example/ch7quiz/MainActivity.java
package com.example.ch7quiz;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void click1(View view){
        Button button1 = findViewById(R.id.button1);

        FragmentManager fm = getSupportFragmentManager();
        FragmentTransaction ft = fm.beginTransaction();
        ft.replace(R.id.content_frame, new Fragment1(), "one");
        ft.commitAllowingStateLoss();
    }

    public void click2(View view){
        Button button2 = findViewById(R.id.button2);

        FragmentManager fm = getSupportFragmentManager();
        FragmentTransaction ft = fm.beginTransaction();
        ft.replace(R.id.content_frame, new Fragment2(), "two");
        ft.commitAllowingStateLoss();
    }
}

File: AndroidStudio_folder/ItemTrade80/app/src/main/java/com/example/itemtrade80/MainActivity.kt
package com.example.itemtrade80

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.Fragment
import com.example.itemtrade80.chat.ChatFragment
import com.example.itemtrade80.home.HomeFragment
import com.example.itemtrade80.myinfo.MyInfoFragment
import com.google.android.material.bottomnavigation.BottomNavigationView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val homeFragment = HomeFragment()
        val myInfoFragment = MyInfoFragment()
        val chatFragment = ChatFragment()

        replaceFragment(homeFragment)
        val bottomNavigationView = findViewById<BottomNavigationView>(R.id.bottom_nav)
        bottomNavigationView.setOnNavigationItemReselectedListener {
            when(it.itemId){
                R.id.page_home -> replaceFragment(homeFragment)
                R.id.page_chat -> replaceFragment(chatFragment)
                R.id.page_myinfo -> replaceFragment(myInfoFragment)
            }
            true
        }
    }

    private fun replaceFragment(fragment: Fragment){
        supportFragmentManager.beginTransaction()
            .apply {
                replace(R.id.fragmentContainer, fragment)
                commit()
            }
    }
}